client/src/pages/Router.tsx

```diff
@@ -46,10 +46,10 @@ const router = createBrowserRouter(
           <Route path="" element={<Reports />} />
           <Route path=":id" element={<ReportDetails />} />
         </Route>
-        <Route path="/settings" element={<Notification />} />
+        <Route path="/notifications" element={<Notification />} />
```

Comment:
pls change that in client/src/components/EntriesSelector/NotificationsEntriesSelector/NotificationsEntriesSelector.tsx line 89 as well

Reply:
K